rlberry/agents/agent.py

```diff
@@ -584,11 +636,8 @@ class AgentTorch(Agent):
     """
 
     def save(self, filename):
+        # Overwrite the 'save' method to manage CPU and GPU with torch agent
```

Comment:
Why this is not in the docstring?

Reply:
I move it outside the docstring, because it's not really useful for the user, only for the developer. But I can put it in back if we think that was better before.